VndbSharp/Converters/AliasesConverter.cs

```diff
@@ -10,13 +11,23 @@ public override void WriteJson(JsonWriter writer, Object value, JsonSerializer s
 			throw new NotImplementedException();
 		}
 
-		public override Object ReadJson(JsonReader reader, Type objectType, Object existingValue, JsonSerializer serializer)
-		{
-			var values = reader.Value?.ToString().Split('\n');
```

Comment:
This could have been done as ```
var values = reader.Value?.ToString().Split(new char[] { '\n', ',' }, StringSplitOptions.RemoveEmptyEntries);```, without throwing any NREs.

Can you also provide a location where ',' is used, to see if it should have a different converter or not.